tests/test_field.py

```diff
@@ -116,6 +116,26 @@ def test_returns_a_string_without_placeholders(content):
             {"warning": False},
             ""
         ),
+        (
+            "Please report to the ((>location)) office at ((&time)) on ((<day)).",
+            {">location": "London", "&time": "09:00", "<day": "Monday"},
+            "Please report to the London office at 09:00 on Monday."
+        ),
+        (
+            "Please report to the ((&gt;location)) office at ((&amp;time)) on ((&lt;day)).",
+            {"&gt;location": "Manchester", "&amp;time": "08:00", "&lt;day": "Thursday"},
+            "Please report to the Manchester office at 08:00 on Thursday."
+        ),
```

Comment:
Can we get a test case with these as unescaped values - which I think is actually the case we're more interested in?

eg just `date&time` in both places rather than `date&amp;time`?

Reply:
👀 you've pointed out the above test, oops